tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/Common/MatterProvision.cs

```diff
@@ -1074,104 +1067,101 @@ internal GenericResponseVM ShareMatter(MatterMetdataVM matterMetadata, string ma
             if (!string.IsNullOrWhiteSpace(mailSiteURL))
             {
                 using (ClientContext clientContext = spoAuthorization.GetClientContext(mailSiteURL))
-                {
-
+                {                  
+                    
                     genericResponse = validationFunctions.MatterDetailsValidation(matter, client,
                         int.Parse(ServiceConstants.ProvisionMatterShareMatter, CultureInfo.InvariantCulture), matterConfigurations);
                     if (genericResponse != null)
                     {
                         return genericResponse;
                     }
 
+                    MatterInformationVM matterInformation = new MatterInformationVM()
+                    {
+                        Client = client,
+                        Matter = matter,
+                        MatterDetails = matterDetails
+                    };
+
                     // Get the current logged in User
                     clientContext.Load(clientContext.Web.CurrentUser);
-                    clientContext.ExecuteQuery();

```

Comment:
This code will serialize the matter to SendMail list which is used to send notification to all assigned users. Now we dont have SendMail list concept and we will be using Azure table storage to store matter information and azure web job will read this matter information to send notification emails